internal/services/event_data_models.go

```diff
@@ -90,3 +90,18 @@ type DeviceDefinition struct {
 		Year  int
 	}
 }
+
+type NewDCNNodeEventData struct {
+	Node  []byte
+	Owner common.Address
+}
+
+type NewDCNResolverEventData struct {
```

Comment:
Let’s actually not handle this. We’ll just look at the default resolver and ignore everything else.